docs/source/bft_configuration.md

```diff
@@ -0,0 +1,223 @@
+* `IncomingMessageBufferSize`: The size of the buffer holding incoming messages before they are processed (maximal number of messages).
+* `RequestPoolSize` : The number of pending requests retained by the node. The `RequestPoolSize` is recommended to be at least double (x2) the `RequestBatchMaxCount`. This cannot be changed dynamically and the node must be restarted to pick up the change.
+* `RequestForwardTimeout`: Is started from the moment a request is submitted, and defines the interval after which a request is forwarded to the leader.
+* `RequestComplainTimeout`: Is started when `RequestForwardTimeout` expires, and defines the interval after which the node complains about the view leader.
+* `RequestAutoRemoveTimeout`: Is started when `RequestComplainTimeout` expires, and defines the interval after which a request is removed (dropped) from the request pool.
+* `ViewChangeResendInterval`: Defines the interval after which the ViewChange message is resent.
+* `ViewChangeTimeout`: Is started when a node first receives a quorum of `ViewChange` messages, and defines the interval after which the node will try to initiate a view change with a higher view number.
+* `LeaderHeartbeatTimeout`: Is the interval after which, if nodes do not receive a "sign of life" from the leader, they complain about the current leader and try to initiate a view change. A sign of life is either a heartbeat or a message from the leader.
+* `LeaderHeartbeatCount`: Is the number of heartbeats per `LeaderHeartbeatTimeout` that the leader should emit. The heartbeat-interval is equal to: `LeaderHeartbeatTimeout/LeaderHeartbeatCount`.
+* `CollectTimeout`: Is the interval after which the node stops listening to `StateTransferResponse` messages, stops collecting information about view metadata from remote nodes.
+
+
+## Block validation policy 
+The block validation policy in a Fabric channel defaults to a policy that requires a signature from any orderer:
+```
+        # BlockValidation specifies what signatures must be included in the block
+        # from the orderer for the peer to validate it.
+        BlockValidation:
+            Type: ImplicitMeta
+            Rule: "ANY Writers"
+```
+
+In BFT, the [configtxgen](./commands/configtxgen.md) tool encodes a policy that is suitable for BFT.
+It automatically derives the policy from the nodes configured in the `ConsenterMapping` section.
+However, when adding or removing ordering service nodes from the channel,
+the policy should be adjusted accordingly as described in the reconfiguration [tutorial](https://github.com/hyperledger/fabric/blob/main/docs/source/create_channel/add_orderer.md#3-add-the-orderer-to-the-policy-rules).
```

Comment:
```suggestion
the policy should be adjusted accordingly as described in the reconfiguration [tutorial](./create_channel/add_orderer.html#add-the-orderer-to-the-policy-rules).
```

We use relative links so that it works regardless of the branch name, this should do it.